ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Mempool.hs

```diff
@@ -757,41 +764,14 @@ instance TxRefScriptsSizeTooBig DijkstraEra where
           , mismatchExpected = limit
           }
 
------
-
-txMeasureBabbage ::
-  forall proto era.
-  ( ShelleyCompatible proto era
-  , L.AlonzoEraTxWits era
-  , L.BabbageEraTxBody era
-  , ExUnitsTooBigUTxO era
-  , MaxTxSizeUTxO era
-  ) =>
-  TickedLedgerState (ShelleyBlock proto era) ValuesMK ->
-  GenTx (ShelleyBlock proto era) ->
-  V.Validation (TxErrorSG era) ConwayMeasure
-txMeasureBabbage st tx@(ShelleyTx _txid tx') =
-  (\x -> ConwayMeasure x refScriptBytes) <$> txMeasureAlonzo st tx
- where
-  utxo = SL.getUTxO $ tickedShelleyLedgerState st
-
-  -- The Babbage rules should have checked this ref script size against a
-  -- limit, but they did not. Now that Cardano @mainnet@ is in Conway, that
-  -- omission is no longer an attack vector. Any other chain intending to
```

Comment:
Out of curiosity. Why is consensus concerned about other chains? Don't we have enough things to worry about for Cardano?

Reply:
You don't see the fact that consensus cannot be build with any ledger, except `cardano-ledger` as a contradiction to this capability of `ouroboros-consensus` capable of operating on "other chains", whatever that means?